ffc-pay-file-consumer/config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  blobConnectionString: process.env.BATCH_STORAGE,
+  shareConnectionString: process.env.DAX_STORAGE,
+  containerName: 'dax',
+  inboundFolder: 'inbound',
+  archiveFolder: 'archive',
```

Comment:
We have this `archiveFolder` var but do not archive the file once completed, only delete

Reply:
Removed